micro-infra-spring-config/src/main/java/com/ofg/infrastructure/property/FileSystemLocator.java

```diff
@@ -77,7 +78,12 @@ public ConfigLocations getConfigLocations() {
     private Map<String, Object> decrypt(Map<String, Object> sourceMap) {
         final Map<String, Object> result = new HashMap<String, Object>();
         for (Map.Entry<String, Object> entry : sourceMap.entrySet()) {
-            result.put(entry.getKey(), decryptIfEncrypted(entry.getValue()));
+            try {
+                result.put(entry.getKey(), decryptIfEncrypted(entry.getValue()));
+            } catch (Exception e) {
+                log.error(String.format("Exception occurred while trying to decrypt key [%s]", entry.getKey()), e);
```

Comment:
As I understand the idea you would like to make it easier to spot that the exception is related to a decryption operation. Maybe it would be enough to log on error with an error type and message and let the stacktrace be logged in by the higher-level exception handler?